web/manifest/manifest.go

```diff
@@ -153,11 +153,16 @@ func (m *Manifest) conversion() http.HandlerFunc {
 
 		// Save the application configuration to the envFile
 		envToUpdate := map[string]string{
-			appID:         strconv.FormatInt(*config.ID, 10),
-			appKey:        appKeyFile,
-			clientID:      *config.ClientID,
-			clientSecret:  *config.ClientSecret,
-			webhookSecret: *config.WebhookSecret,
+			appID:        strconv.FormatInt(*config.ID, 10),
+			appKey:       appKeyFile,
+			clientID:     *config.ClientID,
+			clientSecret: *config.ClientSecret,
```

Comment:
Actually, this should be changed to this:
```go
clientID:      config.GetClientID(),
clientSecret:  config.GetClientSecret(),
webhookSecret: config.GetWebhookSecret(),
```
This should avoid having to check for `nil` since it returns `""` when `nil`.

Reply:
And while already touching this file, we should fix these:
```go
if err := os.WriteFile(appKeyFile, []byte(config.GetPEM()), 0o600); err != nil {
```
```go
appID:         strconv.FormatInt(config.GetID(), 10),
```
```go
	log.Printf("Successfully created the %s GitHub App.", config.GetName())

	data := struct {
		Name string
	}{
		Name: config.GetName(),
	}
```
```go
	if err := env.Save(publicEnvFile, map[string]string{
		"QUICKFEED_APP_URL": config.GetHTMLURL(),
	}); err != nil {
		return err
	}
	log.Printf("App URL saved in %s: %s", publicEnvFile, config.GetHTMLURL())
```